### PR TITLE
[ASM] - EXPANDER 8513 - XDR Mitigation for ASM

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[EXPANDR-8513](https://jira-dc.paloaltonetworks.com/browse/EXPANDR-8513)

## Description

- Cortex ASM - ASM Alert playbook to support Cortex Endpoint remediation option being selected by a user

- Cortex ASM - Remediation Path Rules playbook to incorporate a new option for Cortex Endpoint remediation to be given as an option to users. This option will be prioritized after cloud and on-premise remediation options. Also, **removed the Cortex ASM - Remediation Objectives** playbook from Cortex ASM - Remediation Path Rules playbook.

- New ** Cortex ASM - Cortex Endpoint Remediation** playbook used to isolate a single Cortex Endpoint (XSIAM/XDR) for remediation purposes.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
some changes plus notes in https://jira-dc.paloaltonetworks.com/browse/EXPANDR-8513

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Cortex_Endpoint_Remediation.yml [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+- inputs:
+  - CortexEndpointID
+  name: General (Inputs group)
+  description: Generic group for inputs
+outputSections:
+- outputs: []
+  name: General (Outputs group)
+  description: Generic group for outputs
+outputs: []
+quiet: true
```
[the remark this answers is not in the document]
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Cortex_Endpoint_Remediation.yml:522]: [dbbbce5](https://github.com/demisto/content/pull/33857/commits/dbbbce5cb0f0fe753cce877c86f8c414973bac04)

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 2]
```diff
@@ -139,6 +140,12 @@ This playbook is used to pull information from Cortex Endpoint (XSIAM/XDR) syste
 
 ![Cortex ASM - Cortex Endpoint Enrichment](https://raw.githubusercontent.com/demisto/content/935a77339c2b1ecde3b9ea64992018bd625c61ed/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Cortex_Endpoint_Enrichment.png)
 
+#### Cortex ASM - Cortex Endpoint Remediation
+
+This playbook is used for remediating a single exposed Cortex Endpoint (XSIAM/XDR) by isolating the endpoint from the network using the "Isolate Endpoint" feature in XSIAM ([see XSIAM details](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Isolate-an-Endpoint)) and XDR ([see XDR details](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Isolate-an-Endpoint)). 
+
+![Cortex ASM - Cortex Endpoint Remediation](https://raw.githubusercontent.com/demisto/content/935a77339c2b1ecde3b9ea64992018bd625c61ed/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Cortex_Endpoint_Enrichment.png)
```
[the remark this answers is not in the document]
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/README.md]: [53cecc4](https://github.com/demisto/content/pull/33857/commits/53cecc45c1dc1a564a3468effd3ee152a81aada6)

**Review by johnnywilkes — APPROVED** [review]
@BigEasyJ , thanks for making those changes.  Approved!

**Comment by BigEasyJ** [comment]
@anas-yousef @ssokolovich this PR is ready from the Xpanse side.

**Review by anas-yousef — APPROVED** [review]

**Pull request merged**